library/src/updater.rs

```diff
@@ -320,7 +320,15 @@ fn patch_base(config: &UpdateConfig) -> anyhow::Result<Box<dyn ReadSeek>> {
     Ok(Box::new(base_r))
 }
 
-#[cfg(not(any(target_os = "android", test)))]
+#[cfg(all(target_os = "windows", not(test)))]
+fn patch_base(config: &UpdateConfig) -> anyhow::Result<Box<dyn ReadSeek>> {
```

Comment:
I think File already is ReadSeek, so you can just to `fs::File::open(&config.libapp_path)`